bin/generate-test-app.sh

```diff
@@ -8,6 +8,9 @@ mkdir rails-test-app/spec/models
 mkdir rails-test-app/spec/system
 mkdir rails-test-app/spec/requests
 
+echo "Note! CI is not allowing our build scripts to write to package.json"
+echo "Note! This package.json may need to be updated in the future."
+cp test-app/package.json rails-test-app/package.json
```

Comment:
This is a hacky tradeoff to CI having odd permissions. 

Normally in install, `cssbundling` and `jsbundling` will add lines to the `package.json`. This isn't happening on CI. This line forces the issue for the test app alone, but I still dont' love it, and if the `package.json` changes in the future this will need to be updated.

Reply:
haha welp ok then

Reply:
Oof, yeah this certainly helps to make the case for exploring GH Actions 😅

Reply:
Oddly, github actions show the same issue, and require a manual copy. I'd love to find a less bullish way of doing this. 

`npm set script` *should* be available in modern (< 14) versions of node, but i worry about general availability there as well. 

Also worth mentioning: this is a gem that requires JS dependencies. How do we reflect that?